lib/backends/now/storage.rb

```diff
@@ -0,0 +1,245 @@
+module Backends
+  module Now
+    class Storage < Backends::Now::Base
```

Comment:
No dummies, please. If you need to use a dummy for testing, combine `dummy` backends for `compute` and `storage` with a real `network` backend for NOW.